modules/packages/MPI.chpl

```diff
@@ -189,6 +189,7 @@ MPI Module Documentation
 
 */
 module MPI {
+  import MPI.C_MPI;
```

Comment:
@mppf should correct me if I'm wrong, but I believe we have enabled relative use statements so this could just be `this.C_MPI`.  There's probably other cases as well

Reply:
It could be but I think Brad was using the absolute paths in case we decide to change the syntax